tests/test_eigen.cpp

```diff
@@ -244,6 +245,7 @@ TEST_SUBMODULE(eigen, m) {
 
     // test_fixed, and various other tests
     m.def("fixed_r", [mat]() -> FixedMatrixR { return FixedMatrixR(mat); });
+    // NOLINTNEXTLINE(readability-const-return-type)
     m.def("fixed_r_const", [mat]() -> const FixedMatrixR { return FixedMatrixR(mat); });
```

Comment:
This used to return a non-writable numpy array, now it returns a writable one.

Reply:
*Unchanged files with check annotations (Beta)* - very nice to see!

Reply:
How annoyingly offspec, Fixing this.

Reply:
It's a kind-of-handy and "cute" trick, but not at all "normal" C++!

Reply:
Is that trick explained somewhere? (I still don't know what it is.) A one-line comment here with a hint would be nice.